src/renderer/components/dialog/dialog.tsx

```diff
@@ -92,15 +92,15 @@ export class Dialog extends React.PureComponent<DialogProps, DialogState> {
     this.props.onOpen();
     if (!this.props.pinned) {
       if (this.elem) this.elem.addEventListener('click', this.onClickOutside);
-      window.addEventListener('keydown', this.onEscapeKey);
+      document.body.addEventListener('keydown', this.onEscapeKey);
```

Comment:
Please add comment (to the code) why `document.body` is used and how related to `Drawer`'s event(s).